packages/myst-execute/README.md

```diff
@@ -0,0 +1,3 @@
+# MyST Execute
+
+This library executes MyST content with Jupyter. It provides a transform that take a MyST tree and fills output nodes. It also provides caching for accessing code block outputs by ID.
```

Comment:
This is super basic from our call this morning - please elaborate or rewrite however you see fit.